src/ed_auth/application/features/auth/dtos/verify_token_dto.py

```
@@ -1,5 +1,5 @@
-from typing import TypedDict
+from pydantic import BaseModel
```

Pylint (3.3.7)

[convention] Missing module docstring

(C0114)